lib/index.js

```diff
@@ -146,7 +146,15 @@ exports.init = function(options, callback) {
                 }
             }
             if (data[item]) {
-                sorted[item] = data[item];
+                if (options.onlyunknown) {
+                    if (data[item].licenses && data[item].licenses !== UNKNOWN) {
```

Comment:
Great catch. Somehow I thought the UNKNOWN licenses are converted some something.
A proper check is now added. It will check for an 'UNKNOWN' text now.